openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -384,6 +443,28 @@ Note: When using this profile with other cryptosuites, it is recommended to be e
         </front>
 </reference>
 
+<reference anchor="ISO.18013-7" target="https://www.iso.org/standard/82772.html">
+        <front>
+          <title>ISO/IEC DTS 18013-7 Personal identification — ISO-compliant driving license — Part 7: Mobile driving license (mDL) add-on functions</title>
+          <author>
+            <organization> ISO/IEC JTC 1/SC 17 Cards and security devices for personal identification</organization>
+          </author>
+          <date year="2024"/>
+        </front>
+</reference>
+
```

Comment:
We are missing a reference to 23220-3 (that is used before - line 87), but it is not published yet and shows as deleted on the iso page?

```suggestion

<reference anchor="ISO.23220-3" target="https://www.iso.org/standard/79125.html">
        <front>
          <title>ISO/IEC DTS 23220-3 Cards and security devices for personal identification — Building blocks for identity management via mobile devices</title>
          <author>
            <organization> ISO/IEC JTC 1/SC 17 Cards and security devices for personal identification</organization>
          </author>
          <date year="2023"/>
        </front>
</reference>

```

Reply:
yeah, I don't know why it shows as cancelled, but reluctant to add a reference to a draft that says deleted....